pkg/utils/v1alpha1/kubernetes.go

```diff
@@ -173,7 +190,13 @@ func FetchAndUpdateVolInfos(nodeID string) (err error) {
 	}
 
 	for _, csivol := range csivols.Items {
+		if csivol.DeletionTimestamp != nil {
```

Comment:
instead of non-nil, should we check for nonzero? not sure, maya follows checking for nonzero

Reply:
But DeletionTimestamp is a pointer.

Reply:
yes.. it seems to be set to empty struct.. we can add 'AND' case of non-zero

Reply:
```
time="2019-07-28T14:25:54Z" level=info msg="0.1.0 - 626096fcc0301994be1861b46a0bf817afd7f4bf"
time="2019-07-28T14:25:54Z" level=info msg="DriverName: openebs-csi.openebs.io Plugin: node EndPoint: unix:///plugin/csi.sock URL: $(OPENEBS_CSI_API_URL) NodeID: csi-ubuntu-16"
time="2019-07-28T14:25:54Z" level=info msg="enabling volume access mode: SINGLE_NODE_WRITER"
time="2019-07-28T14:25:54Z" level=info msg="DELTimestamp: <nil>"
time="2019-07-28T14:25:55Z" level=info msg="GRPC call: /csi.v1.Identity/GetPluginInfo"
time="2019-07-28T14:25:55Z" level=info msg="GRPC request: {}"
time="2019-07-28T14:25:55Z" level=info msg="GRPC response: {\"name\":\"openebs-csi.openebs.io\",\"vendor_version\":\"0.1.0\"}"
time="2019-07-28T14:25:55Z" level=info msg="GRPC call: /csi.v1.Node/NodeGetInfo"
time="2019-07-28T14:25:55Z" level=info msg="GRPC request: {}"
time="2019-07-28T14:25:55Z" level=info msg="GRPC response: {\"node_id\":\"csi-ubuntu-16\"}"
time="2019-07-28T14:25:56Z" level=info msg="GRPC call: /csi.v1.Node/NodeGetInfo"
time="2019-07-28T14:25:56Z" level=info msg="GRPC request: {}"
time="2019-07-28T14:25:56Z" level=info msg="GRPC response: {\"node_id\":\"csi-ubuntu-16\"}"
```
Above logs are created on printing deletion timestamp, which indicates it is received as NULL when not set.